bravado/client.py

```diff
@@ -271,15 +282,16 @@ def response_callback(incoming_response, operation):
     raise_on_unexpected(incoming_response)
 
     try:
-        swagger_return_value = unmarshal_response(incoming_response, operation)
+        incoming_response.swagger_result = unmarshal_response(
```

Comment:
now that we are returning back a tuple having separate fields for swagger_result and response, we can do away with polluting `incoming_response` object with `swagger_result`?